autogluon/task/image_classification/image_classification.py

```diff
@@ -27,33 +27,38 @@ class ImageClassification(BaseTask):
     Classifier = Classifier
 
     @staticmethod
-    def Dataset(*args, **kwargs):
-        """Dataset for AutoGluon image classification tasks. 
-           May either be a :class:`autogluon.task.image_classification.ImageFolderDataset`, :class:`autogluon.task.image_classification.RecordDataset`, 
+    def Dataset(path=None, train=True, name=None, input_size=224, crop_ratio=0.875, *args, **kwargs):
+        """Dataset for AutoGluon image classification tasks.
+           May either be a :class:`autogluon.task.image_classification.ImageFolderDataset`, :class:`autogluon.task.image_classification.RecordDataset`,
            or a popular dataset already built into AutoGluon ('mnist', 'fashionmnist', 'cifar10', 'cifar100', 'imagenet').
 
         Parameters
         ----------
+        path : str
```

Comment:
should this say "optional" like it does for the `name` arg?

I think the doc should also state:  "Either `path` or `name` must be specified but not both."
(and I also think these two args should be the first two and train should come later).